customization/custom-data.md

```diff
@@ -27,5 +27,19 @@ While __data.customData__ affords users extensive freedom in including custom co
 * Are there existing Eiffel events and/or event members able to express the information? Using the standard vocabulary and syntax should always be the first option.
 * If your use case lacks support in the standard Eiffel vocabulary, there's a chance this is actually a general use case which deserves such support. Create an [Issue](https://github.com/Ericsson/eiffel/issues) about it! It is always better to design a common solution than to implement multiple local adaptations.
 * Users defining __data.customData__ members are responsible for them and any compatibility issues. Special considerations or support from standard Eiffel events or syntax can not be expected, unless the custom syntax is proposed to and accepted into the standard Eiffel vocabulary (and consequently is no longer custom).
+* Do you introducing custom data to aggregate data on the producer side? If the
```

Comment:
Should this be about not adding data that needs specific contextual knowledge? Or should it be about aggregation?

Reply:
"Do you aggregate data on the producer side and add that data as custom data? If the producer aggregates the data it will do so for a specific consumer but we want to create a protocol that can serve any consumer for use cases that has not yet seen the light. Therefore try to use the existing fields and aggregate data on the consumer side."

Reply:
Fixed in [6d5312c](https://github.com/eiffel-community/eiffel/pull/305/commits/6d5312c07d053524364f139d5805cf1c1c0c60d8)